src/zrc/api/serializers.py

```diff
@@ -361,7 +387,8 @@ def validate(self, attrs):
 
         # validate that all InformationObjects have indicatieGebruiksrecht set
         if validated_attrs['__is_eindstatus']:
-            zios = validated_attrs['zaak'].zaakinformatieobject_set.all()
+            zaak = attrs['zaak']
```

Comment:
hrm, how come this changed from `validated_attrs` to `attrs`?

Reply:
omg. That's obviously a mistake. Thanks for noticing. Fixed.